__tests__/wallet/wallet.test.ts

```diff
@@ -746,4 +751,51 @@ test('sendTransaction', async () => {
       "pin": "1234"
     }
   );
+});
+
+test('start', async () => {
+  const requestPassword = jest.fn();
+  const network = new Network('testnet');
+  const seed = defaultWalletSeed;
+  const store = new MemoryStore();
+  const storage = new Storage(store);
+
+  jest.spyOn(HathorWalletServiceWallet.prototype, 'pollForWalletStatus').mockImplementation(() => Promise.resolve());
+  jest.spyOn(HathorWalletServiceWallet.prototype, 'setupConnection').mockImplementation(jest.fn());
+  jest.spyOn(walletApi, 'getNewAddresses')
+    .mockImplementation(() => Promise.resolve({ success: true, addresses: [] }));
+  jest.spyOn(walletApi, 'createWallet')
+    .mockImplementation(() => Promise.resolve({
+      success: true,
+      status: {
+        walletId: 'id',
+        xpubkey: 'xpub',
+        status: 'creating',
+        maxGap: 20,
+        createdAt: 0,
+        readyAt: 0,
+      },
+    }));
+
+  const wallet1 = new HathorWalletServiceWallet({
+    requestPassword,
+    seed,
+    network,
+    passphrase: '',
+    storage,
+  });
+  await wallet1.start({ pinCode: '1234', password: '1234' });
+  await wallet1.stop();
+
+  // The storage should have the accessData
+  const wallet2 = new HathorWalletServiceWallet({
+    requestPassword,
+    seed,
+    network,
+    passphrase: '',
+    storage,
+  });
+  await wallet2.start({ pinCode: '1234', password: '1234' });
```

Comment:
I feel we should have some assertions here that confirm the storage's access data is being actually read from, and not re-created from scratch.

Reply:
Done